databricks/koalas/indexes.py

```diff
@@ -2247,6 +2247,68 @@ def from_product(iterables, sortorder=None, names=None):
             index=pd.MultiIndex.from_product(iterables=iterables, sortorder=sortorder, names=names)
         ).index
 
+    @staticmethod
+    def from_frame(df, names=None):
+        """
+        Make a MultiIndex from a DataFrame.
+
+        Parameters
+        ----------
+        df : DataFrame
+            DataFrame to be converted to MultiIndex.
+        names : list-like, optional
+            If no names are provided, use the column names, or tuple of column
+            names if the columns is a MultiIndex. If a sequence, overwrite
+            names with the given sequence.
+
+        Returns
+        -------
+        MultiIndex
+            The MultiIndex representation of the given DataFrame.
+
+        See Also
+        --------
+        MultiIndex.from_arrays : Convert list of arrays to MultiIndex.
+        MultiIndex.from_tuples : Convert list of tuples to MultiIndex.
+        MultiIndex.from_product : Make a MultiIndex from cartesian product
+                                  of iterables.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame([['HI', 'Temp'], ['HI', 'Precip'],
+        ...                    ['NJ', 'Temp'], ['NJ', 'Precip']],
+        ...                   columns=['a', 'b'])
+        >>> df  # doctest: +SKIP
```

Comment:
Why skipped?

Reply:
Ah, I thought we don't need this because we're skipping the all tests below.

But I think it's good to not skip this to check if `df` has been created properly.